go/buildhelper/lib.go

```diff
@@ -0,0 +1,43 @@
+package buildhelper
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+var CONNECTION_TIMEOUT = 15 * time.Second
+
+// EthAPI struct
+// based on https://eth.wiki/json-rpc/API
+type EthAPI struct {
+	ApiClient *ethclient.Client
+	retry     int
+	timeout   time.Duration
```

Comment:
@otherview This is unused, since you use the constant directly in `Connect` (maybe you meant to use the struct field value in `Connect` instead?).